lib/apiAiResponseFormatter.js

```diff
@@ -13,24 +13,36 @@ const contextOut = (shownShelterIds) => (
     ] : []
 );
 
-const responseObject = (text, shownShelterIds = []) => (
+const responseObject = (text, shownShelterIds = [], mapsUrl) => (
   {
     speech: text,
     displayText: text,
+    mapsUrl: mapsUrl,
     contextOut: contextOut(shownShelterIds),
     data: {},
     source: 'irma-api',
   }
 );
 
+const googleMapsUrl = (zipLatLon, shelter) => {
+  let mapsUrlOrigin = zipLatLon.join(',');
+  let mapsUrlDestination;
+
+  if (shelter.latitude && shelter.longitude) mapsUrlDestination = `${shelter.latitude},${shelter.longitude}`;
+  else mapsUrlDestination = shelter.address
```

Comment:
Weird syntax, perhaps use a ternary operator here:

```javascript
const mapsUrlDestination =
  (shelter.latitutde && shelter.longitude) ? `${shelter.latitude},${shelter.longitude}` : shelter.address;
```

I think that would make the code a lot more readable.

Reply:
Awesome! Definitely looks better.